modules/recommended-configurable-storage-technology.adoc

```diff
@@ -51,7 +51,7 @@ The following table summarizes the recommended and configurable storage technolo
 
 ^5^ For metrics, using file storage with the `ReadWriteMany` (RWX) access mode is unreliable. If you use file storage, do not configure the RWX access mode on any persistent volume claims (PVCs) that are configured for use with metrics.
 
-^6^ For logging, using any shared storage would be an anti-pattern. One volume per elasticsearch is required.
+^6^ For logging, review the recommended storage solution in configuring persistent storage for the log store section. Using NFS storage as a persistent volume or through NAS such as Gluster can corrupt the data hence NFS is not supported for Elasticsearch storage. You must use one volume per log store.
```

Comment:
NFS is not recommended for Elasticsearch and LokiStack log store in OpenShift Logging. We can mention that in the statement too. For the sentence 'You must use one volume per log store.' do you mean one volume type ?

Reply:
@IshwarKanse It is one volume == one persistent volume as per my understanding

Reply:
@IshwarKanse  Addressed- Thanks for the review